src/software/ai/hl/stp/tactic/move/move_fsm.h

```diff
@@ -29,23 +28,32 @@ struct MoveFSM
     {
         using namespace boost::sml;
 
-        // idle_s and move_s are the two _states_ used in the transition table
-        const auto idle_s = state<idle_state>;
+        // move_s is the _state_ used in the transition table
         const auto move_s = state<move_state>;
 
         // update_e is the _event_ that the MoveFSM responds to
         const auto update_e = event<Update>;
 
-        // this action sets the intent to a move intent corresponding to the update_e
-        // event
+        /**
+         * This is an Action sets the intent to a move intent corresponding to the
```

Comment:
This is an Action *that* sets the intent...

Reply:
fixed